### PR TITLE
Support importing tailscale_dns_preferences resources.

### PR DESCRIPTION
**Pull request by dsymonds**
Also document importing tailscale_acl resources, which was supported from #186.

Partially takes over #125, but only for a resource I'm actively testing. Others can follow in a future PR.

@clstokes @davidsbond

### CODE REVIEW

**Comment by bradfitz** [comment]
I know very little about terraform but nice to see you here, @dsymonds!

**Comment by dsymonds** [comment]
Hey @bradfitz, glad to cross paths with you again!

File: examples/resources/tailscale_acl/import.sh [hunk 1]
```diff
@@ -0,0 +1 @@
+terraform import tailscale_acl.sample_acl acl
```
Comment (knyar) [inline examples/resources/tailscale_acl/import.sh:1]: I believe `tfplugindocs` expects this to match the markdown docs.

```suggestion
# ID doesn't matter.
terraform import tailscale_acl.sample_acl acl
```

File: examples/resources/tailscale_dns_preferences/import.sh [hunk 2]
```diff
@@ -0,0 +1 @@
+terraform import tailscale_dns_preferences.sample_preferences dns_preferences
```
Comment (knyar) [inline examples/resources/tailscale_dns_preferences/import.sh:1]:
```suggestion
# ID doesn't matter.
terraform import tailscale_dns_preferences.sample_preferences dns_preferences
```

**Review by knyar — APPROVED** [review]

**Comment by dsymonds** [comment]
Thanks @knyar, PTAL.

**Comment by knyar** [comment]
Thank you, and sorry that it took so long to get this reviewed!

**Comment by dsymonds** [comment]
All good!

**Pull request merged**